### DISCUSSION TITLE
Upgrade from 2.0.3 to 2.0.4 getting internal Server error

### DISCUSSION THREAD

**Discussion by antoweb** [modoboa/modoboa#3223] [Help]
Ubuntu 20.04
Upgrade modoboda from 2.0.3 to 2.0.4 following the docs https://modoboa.readthedocs.io/en/latest/upgrade.html#specific-upgrade-instructions

After reboot the admin web panel and the webmail too report and errore

Sorry and internal error occurred

**Comment by cantrust-hosting-cooperative** [comment]
Hi @antoweb , to find the exact error you need to enable debug in your Modoboa instance/settings.py:

```
cd /srv/modoboa/instance
cp settings.py settings.bak.py
sed -i -e 's/DEBUG = False/DEBUG = True/' settings.py
systemctl restart uwsgi
```
Now if you go to the page with the error, it will print a detailed error message about what is wrong.  Please paste that back here and maybe it will give some more information.  I found when doing an upgrade that sometimes errors would show up from several versions back in the install instructions, so you might have a previous step that got missed in an upgrade (ie. before 2.0.3).

Please don't leave Debug enalbed after you find the problem, turn it back off again with:

```
cd /srv/modoboa/instance
sed -i -e 's/DEBUG = True/DEBUG = False/' settings.py
systemctl restart uwsgi
```

In production, Modoboa will e-mail the errors to your admin user (instead of them printing in the browser).

**Comment by antoweb** [comment]
Sorry my bad, i had upgrade not in virtual-env
